### PR TITLE
Fix #AR-3945 Readme update as per Arcana Simple Text Readme template

### PR DESCRIPTION
**Pull request by shaloo**
Ref: https://team-1624093970686.atlassian.net/browse/AR-3945 and Readme Template [here](https://team-1624093970686.atlassian.net/wiki/spaces/AN/pages/232751227/Simple+Text+Readme+Template?atlOrigin=eyJpIjoiODQyMzQ3M2M0Yzk3NGIyNGFkM2NmYWViNWQ4ZjBiN2IiLCJwIjoiYyJ9).  Updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,66 @@
...
+
+# Sample Code: Basic Storage and Auth SDK integration
+
+The repository `basic-storage-wallet-integration` contains sample code to demonstrate how you can integrate with Arcana [Auth](https://github.com/arcana-network/auth) and [Storage](https://github.com/arcana-network/storage) SDKs.
+
+It does not have any code for invoking other features and functions of these integrated SDKs.  Check out Arcana [Sample Demo App](https://github.com/arcana-network/demo-app) repository for a full fledged sample dApp that integrates with Arcana SDKs and plugs in Auth and Storage functionalities.
+
+## ⚙️ Installation
+
+### Deploy Software Dependencies
```
Comment (mmjee) [inline README.md]: Deploy? Shouldn't it be something like “Download”?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,66 @@
...
 
-## Project setup
 ```
 yarn install
 ```
 
-### Compiles and hot-reloads for development
+### Run Development Server
```
Comment (mmjee) [inline README.md]: “Run the” ?

File: README.md [hunk 3]
```diff
@@ -1,24 +1,66 @@
...
+
+To re-compile and hot-reload during development, use:
+
 ```
 yarn serve
 ```
 
-### Compiles and minifies for production
+### Production Build
```
Comment (mmjee) [inline README.md]: “Production Build” is an noun phrase, this should be “Build for production”, I believe.

File: README.md [hunk 4]
```diff
@@ -1,24 +1,66 @@
...
+
+You can utilize the source code in this repository for getting started with Arcana Network SDKs. Use these as stand alone basic applications to integrate with the latest Auth and Storage SDK.  You may plug in additional functionality and try out various features of the SDKs.
+
+See [Arcana Documentation](https://docs.beta.arcana.network/) for user guides and SDK Reference Guides.  
+
+To get started with Arcana SDKs quickly, you can also check out other [code samples and templates](https://docs.beta.arcana.network/docs/overview_cs) in the `Developer Resources` section of the documentation.
+
+## 💡 Support
+
+For any queries regarding this repository usage, contact [Arcana support team](mailto:support@arcana.network).
```
Comment (mmjee) [inline README.md]: Shouldn't there be a “the” here? (the Arcana support team)
Plus I think it also needs title casing.

File: README.md (hunk 1 above)
Comment (shaloo) [reply README.md]:
```suggestion
### Download Software Dependencies

File: README.md [hunk 5]
```diff
@@ -1,24 +1,66 @@
...
+
+### Download Software Dependencies
 
-## Project setup
 ```
 yarn install
 ```
 
-### Compiles and hot-reloads for development
+### Run Development Server
```
Comment (shaloo) [inline README.md]:
```suggestion
### Run the Development Server

File: README.md [hunk 6]
```diff
@@ -1,24 +1,66 @@
...
+### Run the Development Server
+
+To re-compile and hot-reload during development, use:
+
 ```
 yarn serve
 ```
 
-### Compiles and minifies for production
+### Production Build
```
Comment (shaloo) [inline README.md]:
```suggestion
### Build for Production

File: README.md [hunk 7]
```diff
@@ -1,24 +1,66 @@
...
+
+You can utilize the source code in this repository for getting started with Arcana Network SDKs. Use it to build a stand alone basic Vue application that integrates with the latest Auth and Storage SDK.  You may plug in additional functionality and try out various features of the SDKs.
+
+See [Arcana Documentation](https://docs.beta.arcana.network/) for user guides and SDK Reference Guides.  
+
+To get started with Arcana SDKs quickly, you can also check out other [code samples and templates](https://docs.beta.arcana.network/docs/overview_cs) in the `Developer Resources` section of the documentation.
+
+## 💡 Support
+
+For any queries regarding this repository usage, contact [Arcana support team](mailto:support@arcana.network).
```
Comment (shaloo) [inline README.md]:
```suggestion
For any queries regarding this repository usage, contact the [Arcana Support Team](mailto:support@arcana.network).

**Comment by mmjee** [comment]
LGTM.

**Pull request merged**